homeassistant/components/azure_data_explorer/strings.json

```diff
@@ -2,17 +2,21 @@
   "config": {
     "step": {
       "user": {
-        "title": "Setup your Azure Data Explorer integration",
-        "description": "Enter connection details.",
         "data": {
-          "cluster_ingest_uri": "Cluster ingest URI",
-          "database": "Database name",
-          "table": "Table name",
+          "cluster_ingest_uri": "Cluster Ingest URI ",
+          "authority_id": "Authority ID",
           "client_id": "Client ID",
           "client_secret": "Client secret",
-          "authority_id": "Authority ID",
-          "use_queued_ingestion": "Use queued ingestion"
-        }
+          "database": "Database name",
+          "table": "Table name",
+          "use_queued_ingestion": "Use Queued ingestion (Mandotory for free cluster)"
```

Comment:
Please use sentence case, I don't believe `Queued` is a brand name or a named feature, but if it is, then it can stay, otherwise make it lowercase